introduction.md

```diff
@@ -3,3 +3,10 @@
 - My name is christian Acuna. Nice to meet you.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?